src/components/booking/ExpandedSessionCard.tsx

```diff
@@ -64,20 +63,19 @@ export const ExpandedSessionCard = ({
       <div className="rounded-b-md bg-gray-500 py-2 text-center">
         <button
           className="w-full capitalize text-white"
```

Comment:
```suggestion
          className="w-full text-white"
```
probably unnecessary since this is hard coded